packages/aurora-erc20/src/bridged-erc20/getAddress.ts

```diff
@@ -0,0 +1,37 @@
+import { getBridgeParams, getNearAccount } from '@near-eth/client/dist/utils'
+import { Account } from 'near-api-js'
+
+/**
+ * Given an erc20 contract address, get the NEAR contract address of the
+ * corresponding BridgeToken contract.
+ *
+ * @param params Uses Named Arguments pattern, please pass arguments as object
+ * @param params.erc20Address Contract address of an ERC20 token on Ethereum
+ * @param params.options Optional arguments.
+ * @param params.options.nep141Factory Bridge token factory account on NEAR.
+ * @returns string Contract address of ERC-20 BridgeToken on Aurora
+ */
+export default async function getAuroraErc20Address (
```

Comment:
Notice this is duplicating code from aurora.js: https://github.com/aurora-is-near/aurora.js/pull/6